docs/reference/release-notes/6.5.asciidoc

```diff
@@ -47,10 +47,61 @@
 [[release-notes-6.5.2]]
 == {es} version 6.5.2
 
-coming[6.5.2]
-
 Also see <<breaking-changes-6.5,Breaking changes in 6.5>>.
 
+[[enhancement-6.5.2]]
+[float]
+=== Enhancements
+
+Aggregations::
+* Handles exists query in composite aggs {pull}35758[#35758]
+
+SQL::
+* Improve validation of unsupported fields {pull}35675[#35675] (issue: {issue}35673[#35673])
+
+[[bug-6.5.2]]
+[float]
+=== Bug fixes
+
+CCR::
+* Only auto follow indices when all primary shards have started {pull}35814[#35814] (issue: {issue}35480[#35480])
+* Avoid NPE in follower stats when no tasks metadata {pull}35802[#35802]
+
+Features/Watcher::
+* Only trigger a watch if new or schedule/changed {pull}35908[#35908]
+
+Infra/Build::
+* Fix jdbc jar pom to not include deps {pull}36036[#36036] (issue: {issue}32014[#32014])
+
+Infra/Packaging::
+* Add missing entries to conffiles  {pull}35810[#35810] (issue: {issue}35691[#35691])
+
+Infra/Scripting::
+* Actually add joda time back to whitelist {pull}35965[#35965] (issue: {issue}35915[#35915])
```

Comment:
I think the script has mixed this up a bit. The "issue" listed here is actually the second PR (below).